apollo-portal/src/main/resources/static/scripts/services/EventManager.js

```diff
@@ -23,24 +23,15 @@ appService.service('EventManager', [function () {
             return;
         }
 
-        if (!context) {
-            context = {};
+        context = context || {};
+
+        if (objectId) {
+            emitEventToSubscribers(eventRegistry[eventType][objectId], context);
+            emitEventToSubscribers(eventRegistry[eventType][ALL_OBJECT]);
```

Comment:
Is it necessary to check `objectId == ALL_OBJECT` here?

Reply:
Exactly, thanks.
Fixed!